evalml/tests/integration_tests/test_nullable_types.py

```diff
@@ -56,7 +56,7 @@ def test_nullable_types_builds_pipelines(
     if automl_algorithm == "iterative":
         pipelines = [pl.name for pl in automl.allowed_pipelines]
     elif automl_algorithm == "default":
-        # TODO: Upon resolution of GH Issue #2691, increase the num of batches.
+        # TODO: Upon resolution of GH Issue #3186, increase the num of batches.
```

Comment:
Changed to the new issue I set up to track adding ensembling support for TS problems.